### ISSUE TITLE
Discovery: Improve alt text by add character limit

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#13105]
## Description
As of the last alt text report (#12397), 16.9% of images had long alt text. We would like to look at ways to improve this by included a character limit pattern to the alt text field.

The suggested pattern that was discussed was a Green/Yellow/Red pattern:
- Green: you are doing well, you are within the suggested character count
- Yellow: you are getting close to the limit
- Red: You have gone over the limit we have suggested HOWEVER we are still going to let you type.
- Hard limit: At a certain point over the soft limit, you have been in the red this whole time, we will put a hard limit in place and you cannot type any more.

## Acceptance Criteria
- [ ] Determine tech feasibility for the suggested Green/Yellow/Red pattern
- [ ] Determine character counts for each limit

### Team
Please check the team(s) that will do this work.

- [x] `CMS Team`
- [ ] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

**Comment by laflannery** [comment]
Closing as this is no longer needed
